Pyrado/pyrado/algorithms/stopping_criteria/rollout_based_criteria.py

```diff
@@ -27,9 +27,11 @@
 # POSSIBILITY OF SUCH DAMAGE.
 
 from abc import abstractmethod
-from typing import Optional
+from typing import List, NoReturn, Optional
 
 import numpy as np
+import scipy as sp
+import scipy.stats
```

Comment:
```suggestion
```
You later just use `sp.stats`

Reply:
Without this import autocompletion from pycharm and so on does not pick up sp.stats because its not loaded.

Reply:
Hm, it works for me but okay